### DISCUSSION TITLE
$USD 2,03 !!! Works : M-Tech UW-01 300Mbps - Realtek RTL8188eu

### DISCUSSION THREAD

**Discussion by Jbmanid** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#266] [Show and tell]
Working perfectly using M-Tech UW-01 USB Wireless Wifi Receiver Adapter 300Mbps - Realtek RTL8188eu on Monterey 12.6.1. This USB Wireless Wifi receiver was very cheap, only $USD 2,03. [Link Picture](https://images.tokopedia.net/img/cache/700/VqbcmM/2022/4/25/578f09dc-db49-49ce-8b95-94dac8b1313c.jpg). Thank you chris1111 for the tools.

**Comment by chris1111** [comment]
Thanks to repport
